test/kitchen/cookbooks/os_prepare/recipes/iptables.rb

```diff
@@ -1,8 +1,10 @@
 if platform_family?("rhel", "debian", "fedora", "amazon", "suse")
-  package value_for_platform_family(
```

Comment:
This is the preferred way to do it, but the platform family should be rhel not centos and oracle which aren't valid platform families